metricbeat/metricbeat.full.yml

```diff
@@ -402,14 +402,10 @@ output.elasticsearch:
   # to.
   #hosts: ["localhost:9092"]
 
-  # The Kafka topic used for produced events. If use_type is set to true, the
```

Comment:
Do we already have some docs about how to use format string? If not, we should definitively create an issue for it.